### DISCUSSION TITLE
Welcome to yii2-starter-kit Discussions!

### DISCUSSION THREAD

**Discussion by trntv** [trntv/yii2-starter-kit#773] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Azam2926** [comment]
Hello everyone, thanks to yii2-starter-kit developers for building such a amazing template 💪

**Comment by vdstud** [comment]
Здравствуйте! У меня такой вопрос: в шаблоне используется yii2-file-kit, в репозитории лежит версия 2.1.5, в которой присутствует опция "useDirindex". Почему в моем проекте в файле vendor/yii2-starter-kit/yii2-file-kit/src/Storage.php нет этой переменной, т.е. используется другая версия файла? "

Команда `composer show -i` показывает `yii2-starter-kit/yii2-file-kit                2.1.5`, однако исходный код другой. Пробовал удалить и устанавить пакет, пробовал создавать новый проект yii2-starter-kit, все равно опции "useDirindex" нету. Скажите, что я делаю не так? Или как использовать пакет без composerа. Спасибо.
